### DISCUSSION TITLE
How can I have a published status for a meeting? I only show Draft or Pending Review

### DISCUSSION THREAD

**Discussion by Benno2024** [code4recovery/12-step-meeting-list#1562] [Q&A]
I am working on the website socalda.org It's for Southern California DA meetings. I want to use your plugin. But when I create a new meeting or edit an existing meeting the status is Draft or Pending Review. How can I make this meeting published so it will show up in the list? I attached 2 pics to show the issue. Any help is much appreciated.
![Screenshot 2024-11-06 at 7 26 52 PM](https://github.com/user-attachments/assets/36efbc6b-d526-41e8-a28b-1c2cd536d23e)
![Screenshot 2024-11-06 at 7 26 29 PM](https://github.com/user-attachments/assets/7f7ea2d6-eb87-44f8-90ff-d9dba8a540f9)

**Comment by joshreisner** [comment]
hi! if it's an in-person meeting did you enter the address? or if it's an online meeting is there an approximate location entered? 

here's what an in-person meeting looks like for me:

![Screenshot 2024-11-06 at 21-01-15 Edit Meeting “Underground Book Study” ‹ Code for Recovery — WordPress](https://github.com/user-attachments/assets/5932a15f-5ea2-4941-baa7-45e7a4347df4)

**Reply by Benno2024** [reply]
Hi Josh Reisner, 
Thank you for getting back to me. It's an online meeting and I put Los Angeles into the Region. And the URL for the zoom meeting. The problem is that at the top right where your meeting says (status published) I only have the option of Draft or Pending Review. As a result the meeting will not show in the list. How can I get the status to say publish.
![Screenshot 2024-11-07 at 8 12 28 AM](https://github.com/user-attachments/assets/5c86607a-e17e-4861-b17d-8b45e3c4f797)
![Screenshot 2024-11-07 at 8 12 08 AM](https://github.com/user-attachments/assets/b13ecd5b-535f-41c8-940b-507c831736b8)

**Reply by joshreisner** [reply]
ak ok! put Los Angeles also as the address. there is a note to this effect on the form, but online meetings need an address, even if it's just a city

**Reply by Benno2024** [reply]
That worked. Thank you I really appreciate your help. 
Looking forward to have all the meeting with you beautiful 12 step meeting wordpress plugin.
